pinax/stripe/models.py

```diff
@@ -252,14 +252,26 @@ def stripe_customer(self):
         )
 
     def __str__(self):
-        return str(self.user)
+        if self.user:
+            return str(self.user)
+        elif self.id:
+            return ", ".join(str(user) for user in self.users.all())
+        return "No User(s)"
 
     def __repr__(self):
-        return "Customer(pk={!r}, user={!r}, stripe_id={!r})".format(
-            self.pk,
-            self.user,
-            str(self.stripe_id),
-        )
+        if self.user:
+            return "Customer(pk={!r}, user={!r}, stripe_id={!r})".format(
+                self.pk,
+                self.user,
+                self.stripe_id,
```

Comment:
use `str(self.stripe_id)` for py2 (and where necessary also)

Reply:
thank you